plugins/Admin/plugin.py

```diff
@@ -359,6 +359,20 @@ def acmd(self, irc, msg, args, commandAndArgs):
             self.Proxy(irc.getRealIrc(), msg, commandAndArgs)
     acmd = wrap(acmd, ['admin', many('something')])
 
+    class sasl(callbacks.Commands):
+
+        @internationalizeDocstring
+        def start(self, irc, msg, args):
+            """takes no arguments
+
+            Manually initiates SASL authentication
+            """
+            if 'sasl' in irc.state.capabilities_ack:
+                irc.startSasl(msg)
+            else:
+                irc.requestCapabilities(['sasl'])
```

Comment:
Let's just add it to `REQUEST_CAPABILITIES` unconditionally, so you can return an error here if it's not negotiated.

Reply:
I would recommend renaming this to something like `sasl auth` or `sasl login`. `start` sounds like something that starts / enables a long running task, which isn't really the case here. (There's no corresponding `stop` command for it, for instance)